File: 2/2 Progny Java/2/Circle.java
public class Circle {
  double x;
  double y;
  double r;

  public Circle(double x, double y, double r) {
    this.x = x;
    this.y = y;
    this.r = r;
  }

  public void enlarge(double f) {
    this.r *= f;
  }

  public double getArea() {
    return this.r * this.r * Math.PI;
  }

  @Override
  public String toString() {
    return "(" + this.x + ", " + this.y + ")" + " - " + this.r;
  }

  public static void main(String[] args) {
    Circle c = new Circle(0, 0, 2);
    
    System.out.println("area = " + c.getArea());
    c.enlarge(2);

    System.out.println("scale by 2 = " + c);
    System.out.println("area = " + c.getArea());
  }
}

File: 3/3 Webprog/2/circle.js
const output = document.getElementById('result')
const num = document.getElementById('num')
const btn = document.getElementById('btn')
btn.addEventListener('click', calculateArea)

function calculateArea() {
  const r = Number(num.value)
  const area = Math.PI * r * r
  output.innerHTML = `The area of a circle with radius ${r} is ${area.toFixed(2)}`
}
